wonderland.js

```diff
@@ -2,16 +2,20 @@ import { tiny, defs } from "./external/common.js";
 import { Shape_From_File } from "./external/obj-file-demo.js";
 import { Custom_Movement_Controls } from "./custom-movement.js";
 
-const { Cube } = defs;
+const { Cube, Torus } = defs;
+const {
+    Vector, Vector3, vec, vec3, vec4, color, hex_color, Shader, Matrix, Mat4, Light, Shape, Material, Scene, Texture,
```

Comment:
where are you using `hex_color`?

Reply:
I thought I'd add all the ones from prev projects bc we might need it soon, but I can delete the ones that haven't been used yet if you'd like.